src/encoding/text.rs

```diff
@@ -38,7 +38,7 @@ use std::collections::HashMap;
 use std::io::Write;
 use std::ops::Deref;
 
-pub use prometheus_client_derive_text_encode::*;
+pub use prometheus_client_derive_encode::*;
```

Comment:
Would it be good to move this line to `src/encoding.proto` so that consumers import Encode trait as follows?:

```diff
- use prometheus_client::encoding::text::Encode;
+ use prometheus_client::encoding::Encode;
```

Reply:
That makes sense to me. Good idea.

Reply:
Done via https://github.com/prometheus/client_rust/pull/83/commits/d992badbe2aaecd29a3de82110d3fb9fade3d24b and bumped up the minor version since that is a breaking change. https://github.com/prometheus/client_rust/pull/83/commits/8fce75651c11705fbbfd31a93f18ea75d4d71c06